metricbeat/Dockerfile

```diff
@@ -1,4 +1,5 @@
 FROM golang:1.21.8
+COPY --from=docker:26.0.0-alpine3.19 /usr/local/bin/docker  /usr/local/bin/
```

Comment:
Can we leave a comment (with a TODO perhaps that it will be addressed via https://github.com/elastic/beats/pull/38678) ?